### PR TITLE
add new preprints

### PR DESCRIPTION
**Pull request by cgreene**

### CODE REVIEW

**Comment by cgreene** [comment]
@miltondp and @jjc2718 : are there any particular images that you'd want to use as a thumbnail for these? I have been dropping them in #12 and then adding the image - feel free to make suggestions so we can get the page updated

File: _data/citations.yaml [hunk 1]
```diff
@@ -1,5 +1,40 @@
...
+    the utility and consistency of gene expression and DNA methylation
+  authors:
+  - Jake Crawford
+  - Brock C. Christensen
+  - Maria Chikina
+  - Casey S. Greene
+  publisher: Cold Spring Harbor Laboratory
+  date: '2021-10-28'
+  link: https://doi.org/gnbdm9
+- id: 10.1101/2021.10.21.21265225
```
Comment (jjc2718) [inline _data/citations.yaml]:
```suggestion
  link: https://doi.org/gnbdm9
  image: https://user-images.githubusercontent.com/2345877/140061720-96b5451a-c9af-493f-85b2-cc4c723d316b.png
- id: 10.1101/2021.10.21.21265225
```

File: _data/citations.yaml [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+  - Krzysztof Kiryluk
+  - Iftikhar Kullo
+  - Yuan Luo
+  - Milton Pividori
+  - Hae Kyung Im
+  - Casey S. Greene
+  - Marylyn D. Ritchie
+  publisher: Cold Spring Harbor Laboratory
+  date: '2021-10-26'
+  link: https://doi.org/gnbdnb
```
Comment (miltondp) [inline _data/citations.yaml:38]:
```suggestion
  link: https://doi.org/gnbdnb
  image: https://user-images.githubusercontent.com/172687/140064066-677bad73-ae37-4968-aa77-ad854b6c8251.png
```

**Comment by cgreene** [comment]
Just realized y'all are updating the auto generated file. Can you suggest in sources so it doesn't get overwritten?

File: _data/sources.yaml [hunk 3]
```diff
@@ -1,3 +1,7 @@
+- id: doi:10.1101/2021.10.27.466140
```
Comment (jjc2718) [inline _data/sources.yaml:1]:
```suggestion
- id: doi:10.1101/2021.10.27.466140
  image: https://user-images.githubusercontent.com/2345877/140061720-96b5451a-c9af-493f-85b2-cc4c723d316b.png
```

File: _data/sources.yaml [hunk 4]
```diff
@@ -1,3 +1,8 @@
...
+  image: https://user-images.githubusercontent.com/2345877/140061720-96b5451a-c9af-493f-85b2-cc4c723d316b.png
+
+- id: 10.1101/2021.10.21.21265225
```
Comment (miltondp) [inline _data/sources.yaml:4]:
```suggestion
- id: 10.1101/2021.10.21.21265225
  image: https://user-images.githubusercontent.com/172687/140064066-677bad73-ae37-4968-aa77-ad854b6c8251.png
```

**Pull request merged**